ui/externs/ui.js

```diff
@@ -357,3 +357,49 @@ shaka.extern.IUISettingsMenu = class {
     this.backSpan;
   }
 };
+
+/**
+ * Interface for SeekBars. SeekBars should inherit from the concrete base
+ * class shaka.ui.Element.  
+ *
+ * @extends {shaka.extern.IUIElement }
```

Comment:
style: remove the space inside the curly braces

Reply:
This one was not reported by the linter, but sure it's ugly now that you pointed it out : )